sw/airborne/arch/chibios/mcu_periph/uart_arch.c

```diff
@@ -128,7 +128,7 @@ static __attribute__((noreturn)) void thd_uart1_rx(void *arg)
   }
 }
 
-static THD_WORKING_AREA(wa_thd_uart1_rx, 1024);
+static THD_WORKING_AREA(wa_thd_uart1_rx, 256);
```

Comment:
dtto - board specific defines for the thread sizes if needed to be smaller than defaults